app/models/sidebar_url.rb

```diff
@@ -5,6 +5,71 @@ class SidebarUrl < ActiveRecord::Base
   MAX_ICON_LENGTH = 40
   MAX_NAME_LENGTH = 80
   MAX_VALUE_LENGTH = 200
+  COMMUNITY_SECTION_LINKS = [
+    {
+      id: -1,
```

Comment:
I'm curious about the reason we need to use negative ids. I think we should try and avoid this if possible.

Reply:
I wanted to emphasize that those are system sections, but I will change it if you think it is a bad idea

Reply:
I don't think there is a need to mess with the ID manually for now since system sections already has a column that acts as an identifier.